cmd/serverless/api_key.go

```diff
@@ -120,3 +129,19 @@ func readAPIKeyFromSecretsManager(arn string) (string, error) {
 	log.Warn("Secrets Manager returned something but there seems to be no data available")
 	return "", nil
 }
+
+func extractRegionFromSecretsManagerArn(secretsManagerArn string) (string, error) {
+	arnObject, err := arn.Parse(secretsManagerArn)
+	if err != nil {
+		return "", fmt.Errorf("could not extract region from arn: %s. %s", secretsManagerArn, err)
+	}
+
+	regionRegex := `\w*-\w*-\d{1}`
```

Comment:
I'd feel more comfortable splitting this by `:` and taking the fourth element, which is guaranteed to be the region.